packages/core/src/internal/decorators/i18n.ts

```diff
@@ -96,3 +110,45 @@ export function i18n() {
       : standardI18n(descriptor, protoOrDescriptor);
   };
 }
+
+export function getI18nValues(values: Record<string, unknown>, component: I18nElement): Record<string, unknown> {
+  if (isNilOrEmpty(values)) {
+    let returnVal = {};
+
+    if (component.hasAttribute('cds-i18n')) {
```

Comment:
This block here is a sledgehammer to force a lookup on the HTML attr in the event the i18n property gets in a bad state. It does no harm but handles a weird edge case I was hitting early on.